docs/specs/moniker.yml

```diff
@@ -240,3 +240,164 @@ outputs:
   9d4e15fd/docs/a.json:
   build.log: |
     ["warning","empty-monikers","No intersection between zone and file level monikers. The result of zone level range string 'netcore-1.2' is 'netcore-1.2', while file level monikers is 'netcore-1.0,netcore-1.1'.","docs/v1/a.md"]
+---
+# Define uid with moniker range, reference to it with moniker in the same docset
+inputs:
+  docfx.yml: |
+    monikerRange:
+      'docs/v1/**/*.md': '< netcore-2.0'
+      'docs/v2/**/*.md': '>= netcore-2.0'
+    routes:
+      docs/v1/: docs/
+      docs/v2/: docs/
+    monikerDefinition: monikerDefinition.json
+  docs/v1/a.md: |
+    ---
+    uid: a
+    title: netcore-1.1
+    monikerRange: '> netcore-1.0'
+    ---
+    Moniker: netcore-1.1
+  docs/v2/a.md: |
+    ---
+    uid: a
+    title: netcore-2.0
+    monikerRange: '<= netcore-2.0'
+    ---
+    Moniker: netcore-2.0
+  docs/a.md: |
+    No version
+  docs/b.md: |
+    Link to @a?view=netcore-1.1
```

Comment:
>netcore-1.1 [](start = 20, length = 11)

if this moniker is invalid?

Reply:
Good point, it should fall back to the latest one, added a e2e test for it